src/plugins/orangehrmRecruitmentPlugin/Controller/VacancyListRestController.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Recruitment\Controller;
+
+use OrangeHRM\Core\Api\CommonParams;
+use OrangeHRM\Core\Api\V2\Exception\NotImplementedException;
+use OrangeHRM\Core\Api\V2\Request;
+use OrangeHRM\Core\Api\V2\Response;
+use OrangeHRM\Core\Api\V2\Validator\Helpers\ValidationDecorator;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Controller\PublicControllerInterface;
+use OrangeHRM\Core\Controller\Rest\V2\AbstractRestController;
+use OrangeHRM\Core\Exception\SearchParamException;
+use OrangeHRM\Core\Traits\Service\NormalizerServiceTrait;
+use OrangeHRM\ORM\ListSorter;
+use OrangeHRM\Recruitment\Api\Model\VacancyListModel;
+use OrangeHRM\Recruitment\Dto\VacancySearchFilterParams;
+use OrangeHRM\Recruitment\Traits\Service\VacancyServiceTrait;
+
+/**
+ *
+ */
+class VacancyListRestController extends AbstractRestController implements PublicControllerInterface
+{
+    use VacancyServiceTrait;
+    use NormalizerServiceTrait;
+
+    public const DEFAULT_STATUS = 1;
+
+    private const VACANCY_ID = 'vacancy.id';
+    private const VACANCY_OFFSET = 'offset';
+    private const VACANCY_LIMIT = 'limit';
+    /**
+     * @var ValidationDecorator|null
+     */
+    private ?ValidationDecorator $validationDecorator = null;
+    /**
+     * @var int
+     */
+    private int $default_status = self::DEFAULT_STATUS;
```

Comment:
```suggestion
    private int $defaultStatus = self::DEFAULT_STATUS;
```
please go with camelCase

Reply:
Sorry fort the typo. Done